kuma/static/js/analytics.js

```diff
@@ -1,8 +1,20 @@
 (function(win, doc, $) {
     'use strict';
 
+    // prettier-ignore
```

Comment:
I am not comfortable committing code that is specific to one contributor's current editor set up :/